jerry-core/ecma/operations/ecma-regexp-object.c

```diff
@@ -1142,16 +1142,8 @@ re_match_regexp (re_matcher_ctx_t *re_ctx_p, /**< RegExp matcher context */
         }
         return ECMA_VALUE_FALSE; /* fail */
       }
-      default:
-      {
-        JERRY_TRACE_MSG ("UNKNOWN opcode (%u)!\n", (unsigned int) op);
-        return ecma_raise_common_error (ECMA_ERR_MSG ("Unknown RegExp opcode."));
```

Comment:
I think this isn't a good decision to change this runtime error to an assert.

Reply:
Please explain. This is the virtual machine of the regex engine executing the opcodes that can only come from "us", i.e., there is no legitimate way for a user to pass in arbitrary byte sequence. If we generate regex byte code propery and interpret it properly, too, then we should not possibly find an unknown opcode here ever. We are not preparing the "main" VM for unknown opcode either. If anything like that happens then we have a design flaw. If I'm not misunderstanding something.

Reply:
Fine, good to me.